install.sh

```diff
@@ -4028,44 +4067,53 @@ install_hooks_into_repo() {
     done
 
     # Offer to setup the intro README if running in interactive mode
-    if is_non_interactive; then
-        true # Let's skip this in non-interactive mode to avoid polluting the local repos with README files
+    # Let's skip this in non-interactive mode or in a bare repository
+    # to avoid polluting the repos with README files
+    if ! is_non_interactive && [ "${IS_BARE}" != "true" ]; then
```

Comment:
Nit: this chain is getting a bit big, it would be good to change to something like
```
if is_non_interactive; then
  true
elif [ "${IS_BARE}" != "true" ]; then
  true
else
  ...
fi
```
I'm happy to do this one after merging if you don't have time, or not at a computer.